app/cdap/components/WranglerGrid/SelectColumnPanel/DrawerHeader/index.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Container, IconButton } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import CloseRoundedIcon from '@material-ui/icons/CloseRounded';
+import styled from 'styled-components';
+import { FlexAlignCenter, PointerBox } from 'components/common/BoxContainer';
+import { HeadFont } from 'components/common/TypographyText';
+import T from 'i18n-react';
+import {
+  ADD_TRANSFORMATION_PREFIX,
+  MULTI_SELECTION_COLUMN,
+} from 'components/WranglerGrid/SelectColumnPanel/constants';
+import grey from '@material-ui/core/colors/grey';
+import ChevronLeftRoundedIcon from '@material-ui/icons/ChevronLeftRounded';
+import { blue } from '@material-ui/core/colors';
+import { IMultipleSelectedFunctionDetail } from '../types';
+
+interface IDrawerHeaderProps {
+  closeClickHandler: () => void;
+  transformationName: string;
+}
+
+export const UnderLine = (
```

Comment:
You've been asked to remove this underline many times at this point.

Reply:
Done

Reply:
removed the underline completely according to V3 design expectations